### ISSUE TITLE
Can it support plot in base package or other packages?

### ISSUE THREAD

**Issue by skyroam** [thebioengineer/camcorder#48]
Thanks for so good a tool. But can it support more plots in other packages later?

**Comment by thebioengineer** [comment]
Yes, so it cannot directly snap the base plots, but `record_polaroid()` can save a snap for you. See https://thebioengineer.github.io/camcorder/articles/record_polaroid.html for a vignette on how to do this
